code/modules/customitems/item_defines.dm

```diff
@@ -731,7 +758,7 @@
 	sprite_sheets = null
 
 /obj/item/clothing/suit/jacket/fluff/kidosvest // Anxipal: Kido Qasteth
-	name = "Kido's Vest"
+	name = "Kido's vest"
```

Comment:
Don't change other fluff items needlessly- The above is fine due to fixing a spelling error, but there's no reason to change this capitalization.